concepts/sepsis3/abx-micro-prescription.sql

```diff
@@ -0,0 +1,140 @@
+DROP TABLE IF EXISTS `physionet-data.mimic_derived.abx_micro_poe`;
+CREATE TABLE `physionet-data.mimic_derived.abx_micro_poe` as
+with abx as
+(
+  select pr.hadm_id
+  , pr.drug as antibiotic_name
+  , pr.starttime as antibiotic_time
+  , pr.stoptime as antibiotic_endtime
+  from `physionet-data.mimic_hosp.prescriptions` pr
+  -- inner join to subselect to only antibiotic prescriptions
+  inner join `physionet-data.mimic_derived.abx_poe_list` ab
+      on pr.drug = ab.drug
+)
+-- get cultures for each icustay
```

Comment:
think I left a comment here that can be deleted

Reply:
Done